examples/guestbook/README.md

```diff
@@ -46,7 +54,7 @@ Use the file `examples/guestbook/redis-master.json` which describes a single pod
 Create the redis pod in your Kubernetes cluster by running:
 
 ```shell
-$ cluster/kubectl.sh create -f examples/guestbook/redis-master.json
+$ cluster/kubectl.sh create -f examples/guestbook/redis-master-controller.json
 ```
 
```

Comment:
Also show that using `cluster/kubectl.sh get rc` will list the replication controller.